Ochowo/iReporter
# connect endpoints from  dummy db to database

**What does this PR do?**
- Connects endpoints from challenge 2 to database
- Create more endpoints

**Description of the task  to be completed**
- Ensure the endpoints below are working as they should
    `POST/api/v1/auth/signup`
    `POST/api/v1/auth/login`
     `POST/api/v1/red-flags`
     `POST/api/v1/auth/interventions`
     `GET/api/v1/redflags`
     `GET/api/v1/interventions`
     `GET/api/v1/redflags/:id`
     `GET/api/v1/interventions/:id`
     `PATCH/api/v1/red-flags/:id/location`
      `PATCH/api/v1/interventions/:id/location`
      `PATCH/api/v1/red-flags/:id/status`
      `PATCH/api/v1/interventions/:id/status`
      `PATCH/api/v1/red-flags/:id/comment`
      `PATCH/api/v1/interventions/:id/comment`
      `DELETE/api/v1/red-flags/:id`
      `DELETE/api/v1/interventions/:id`

**How should this be tested manually**
   After you clone the repo, cd into it and run `npm install` to install node modules. Make sure you have 
   nodejs and express installed. Afterwards, run `npm run start:dev` to initiate the app.
- test the endpoint using postman with `localhost:5000` as the host
- send empty inputs to test validation

**What are the relevant pivotal stories**
- N/A

**Any background context you want to add?**
- N/A

**Screenshots**
N/A
